proposal.md

```diff
@@ -1,31 +1,31 @@
 # Proposal
 
-When a user says:
-My bot will respond by:
+When a user says: tell me a sports score
+My bot will respond by: looking at espn.com and giving that score back
```

Comment:
Let's instead just have the bot tell the user to go look up scores as espn.com. You'll learn web scraping later in this class, so you can come back to this idea later.